qcodes/instrument_drivers/QDev/QDac_channels.py

```diff
@@ -354,35 +375,33 @@ def _set_vrange(self, chan, switchint):
             self._update_v_validator(channel, switchint)
             channel.v.cache.set(QDac._get_v_exp_from_v_dac(channel, v_dac))
 
-    def _get_vrange(self, chan):
+    def _get_vrange(self, chan: int) -> float:
         """
         get_cmd for the chXX_vrange parameter
 
         Args:
-            chan (int): The 1-indexed channel number
+            chan: The 1-indexed channel number
         """
         self._update_cache(readcurrents=False)
         return self.channels[chan - 1].vrange.cache()
 
-
-
-    def _num_verbose(self, s):
+    def _num_verbose(self, s: str) -> float:
         """
         turn a return value from the QDac into a number.
         If the QDac is in verbose mode, this involves stripping off the
         value descriptor.
         """
         if self.verbose.get_latest():
-            s = s.split[': '][-1]
+            s = s.split(': ')[-1]
```

Comment:
is this correct. The original code looks like it never worked?

Reply:
Yeah, `split[...]` do definitely not work.